aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -347,5 +347,45 @@ Tensor& matmul_out(Tensor &result, const Tensor & tensor1, const Tensor & tensor
   return result;
 }
 
+Tensor frobenius_norm(const Tensor& self) {
+  IntList dim = {0, 1};
+  return at::native::frobenius_norm(self, dim);
+}
+
+Tensor frobenius_norm(const Tensor& self, IntList dim, bool keepdim) {
+  Tensor result = self.type().tensor();
+  return at::native::frobenius_norm_out(result, self, dim, keepdim);
```

Comment:
You are breaking autograd here. _out functions are not supported for autograd. You can try to back prop via frobenius_norm. it should break.

Reply:
It seems that current torch.norm seems used a similar way to implement. I am wondering why that works and how we can solve this issue.

See pytorch/aten/src/ATen/native/ReduceOps.cpp line 587
Tensor norm(const Tensor& self, Scalar p, int64_t dim, bool keepdim) {
  Tensor result = self.type().tensor();
  return at::native::norm_out(result, self, p, dim, keepdim);
}

Reply:
Because norm has a custom backward defined in derivatives.yaml, but here you are relying on autograd to run backward for you.